src/workbooks/governance/workbook.json

```diff
@@ -6769,18 +6904,15 @@
                         "version": "KqlParameterItem/1.0",
                         "name": "Subscription",
                         "type": 6,
-                        "isRequired": true,
                         "typeSettings": {
-                          "additionalResourceOptions": [
-                            "value::1"
-                          ],
+                          "additionalResourceOptions": [],
                           "includeAll": true,
                           "showDefault": false
                         },
                         "timeContext": {
                           "durationMs": 86400000
                         },
-                        "value": null
+                        "value": "/subscriptions/3d2171d3-2b1e-41d6-a62c-dc780b0c5a04"
```

Comment:
Do you want to include subscription IDs?

Reply:
hum nop thanks